### PR TITLE
Mr departments create

### PR DESCRIPTION
**Pull request by MattRowee**
Testing and Functionality
---------------------------
When viewing the list of departments
And you click the Create New link
Then a form will be displayed on which the following information can be entered

name
budget

-deleted DepartmentCreateViewModel after review
-added comments to explain functionality of complex code

Checklist:
yes -- I have commented my code, particularly in hard-to-understand areas
no -- I have made corresponding changes to the documentation
yes -- My changes generate no new errors
no -- I have added tests that prove my fix is effective or that my feature works
no -- New and existing unit tests pass locally with my changes

### CODE REVIEW

File: BangazonWorkforce/Models/ViewModels/DeptCreateViewModel.cs [hunk 1]
```diff
@@ -0,0 +1,71 @@
+﻿//using Microsoft.AspNetCore.Mvc.Rendering;
+//using System.Collections.Generic;
+//using System.Data.SqlClient;
+//using System.Linq;
+
+
+//namespace BangazonWorkforce.Models.ViewModels
+//{
+//    public class DeptCreateViewModel
```
Comment (ColonelSped) [inline BangazonWorkforce/Models/ViewModels/DeptCreateViewModel.cs]: What's this doing?

File: BangazonWorkforce/Models/ViewModels/DeptCreateViewModel.cs [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+//    {        
+//        public Department department { get; set; }
+
+//        protected string _connectionString;
+
+//        public DeptCreateViewModel Connection
```
Comment (ColonelSped) [inline BangazonWorkforce/Models/ViewModels/DeptCreateViewModel.cs]: And this?

File: BangazonWorkforce/Models/ViewModels/DeptCreateViewModel.cs [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+//        {
+//            get
+//            {
+//                return new SqlConnection(_connectionString);
+//            }
+//        }
+
+//        public DeptCreateViewModel() { }
```
Comment (ColonelSped) [inline BangazonWorkforce/Models/ViewModels/DeptCreateViewModel.cs]: and this?

File: BangazonWorkforce/Models/ViewModels/DeptCreateViewModel.cs [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+//                .ToList();
+
+//            Cohorts.Insert(0, new SelectListItem
+//            {
+//                Text = "Choose cohort...",
+//                Value = "0"
+//            });
+//        }
+
+//        private List<Cohort> GetAllCohorts()
```
Comment (ColonelSped) [inline BangazonWorkforce/Models/ViewModels/DeptCreateViewModel.cs]: What is this code doing?

**Review by ColonelSped — CHANGES_REQUESTED** [review]

**Comment by Downeastrussell** [comment]
👍

**Review by ColonelSped — APPROVED** [review]

**Comment by ColonelSped** [comment]
👍

**Pull request merged**